src/main/java/com/synopsys/arc/jenkinsci/plugins/extratoolinstallers/installers/RarExtractionInstaller.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009, Sun Microsystems, Inc.
```

Comment:
Like I said, I could use some help with how to correctly update a license as I haven't done this before. Should I add myself next to existing names in the license? Do I even change anything?
What would you do? :)